cypress/e2e/login.cy.js

```diff
@@ -2,10 +2,32 @@
 
 describe('Login page', () => {
   beforeEach(() => {
-    
+    cy.visit('https://the-internet.herokuapp.com/login');
   });
 
-  it('', () => {
-    
+  it('Login with valid creds', () => {
+    cy.get('#username').type("tomsmith");
+    cy.get('#password').type("SuperSecretPassword!");
+    cy.get('.fa').click();
+    cy.wait(4000);
+    cy.get('h4.subheader').should('have.text','Welcome to the Secure Area. When you are done click logout below.');
```

Comment:
```suggestion
    cy.get('h4.subheader', { timeout: 4000 }).should('have.text','Welcome to the Secure Area. When you are done click logout below.');
```
will tell the difference tomorrow, pls remind me about it in case I'll forget